docs/usage/files.md

```diff
@@ -217,6 +217,28 @@ print('File "{0}" uploaded to Box with file ID {1}'.format(uploaded_file.name, u
 [get_chunked_uploader]: https://box-python-sdk.readthedocs.io/en/latest/boxsdk.object.html#boxsdk.object.upload_session.UploadSession.get_chunked_uploader
 [get_chunked_uploader_for_stream]: https://box-python-sdk.readthedocs.io/en/latest/boxsdk.object.html#boxsdk.object.upload_session.UploadSession.get_chunked_uploader_for_stream
 
+#### Resume Upload
+
+Sometimes an upload can be interrupted, in order to resume uploading where you last left off, simply call the
+[`chunked_uploader.resume()`][resume] method. This will return the the [File][file_class] object that was uploaded.
+
+```python
```

Comment:
We should probably flesh out this example, since it doesn't really make sense as written — why would the user call resume on a new `ChunkedUploader`?  Maybe show a try/except with a call to `start()` that raises an error and a call to `resume()` in the except block?